providers/fips.module.sources

```diff
@@ -434,34 +393,21 @@ providers/implementations/keymgmt/ecx_kmgmt.c
 providers/implementations/keymgmt/kdf_legacy_kmgmt.c
 providers/implementations/keymgmt/mac_legacy_kmgmt.c
 providers/implementations/keymgmt/rsa_kmgmt.c
-providers/implementations/macs/blake2b_mac.c
-providers/implementations/macs/blake2s_mac.c
 providers/implementations/macs/cmac_prov.c
 providers/implementations/macs/gmac_prov.c
 providers/implementations/macs/hmac_prov.c
 providers/implementations/macs/kmac_prov.c
-providers/implementations/macs/poly1305_prov.c
-providers/implementations/macs/siphash_prov.c
 providers/implementations/rands/crngt.c
 providers/implementations/rands/drbg.c
 providers/implementations/rands/drbg_ctr.c
 providers/implementations/rands/drbg_hash.c
 providers/implementations/rands/drbg_hmac.c
-providers/implementations/rands/seed_src.c
-providers/implementations/rands/seeding/rand_cpu_x86.c
-providers/implementations/rands/seeding/rand_tsc.c
-providers/implementations/rands/seeding/rand_unix.c
-providers/implementations/rands/seeding/rand_win.c
 providers/implementations/rands/test_rng.c
-providers/implementations/signature/dsa.c
-providers/implementations/signature/ecdsa.c
-providers/implementations/signature/eddsa.c
-providers/implementations/signature/mac_legacy.c
-providers/implementations/signature/rsa.c
-providers/implementations/signature/sm2sig.c
-providers/implementations/storemgmt/file_store.c
-providers/implementations/storemgmt/file_store_der2obj.c
-providers/prov_running.c
+providers/implementations/signature/dsa_sig.c
```

Comment:
wondered what the heck was going on here for a sec.. You renamed files :)

Reply:
Yup